templates/README.md

```diff
@@ -0,0 +1,6 @@
+# Ably Templates
+
+| Template | Target | Target Name | Synopsis |
```

Comment:
Minor suggestion. Would the following titles be simpler?

Template | Repository | Filename | Description

Reply:
I like it. I'll push a commit. Thanks.

Reply:
Done: https://github.com/ably/ably-common/pull/88/commits/ad0cbdab7d97dd8cb8e0f4badc92fe3efbc8b9bb